sycl/doc/developer/DockerBKMs.md

```diff
@@ -143,10 +143,20 @@ instructions.
 
 ## Changing Docker user
 
-By default all processes inside Docker run as root. Some LLVM or Clang tests
-expect your user to be anything but root. You can change the user by specifying
-`-u <username or uid>` option. All Docker containers come with user `sycl`
-created.
+By default all processes within our containers are run as `sycl_ci` user.
```

Comment:
```suggestion
By default all processes within our containers are run as the `sycl_ci` user.
```

Reply:
Applied in ce80d93eb584ab2104129170103a275429176e48